Packs/AlibabaActionTrail/Integrations/AlibabaActionTrailEventCollector/README.md

```diff
@@ -7,18 +7,18 @@ This integration was integrated and tested with API version 0.6 of Alicloud Log
 2. Search for Alibaba Action Trail Event Collector.
 3. Click **Add instance** to create and configure a new integration instance.
 
-    | **Parameter** | **Required** |
-    | --- | --- |
-    | Endpoint | True |
-    | Access key id | True |
-    | Access key | True |
-    | Project name | True |
-    | Logstore name | True |
-    | Query | True |
-    | Number of incidents to fetch per fetch. | False |
-    | First fetch time interval | False |
-    | Use system proxy settings | False |
-    | Use Secured Connection | False |
+    | **Parameter** | **Description** | ** **Required** |
+    | --- | --- | --- |
+    | Endpoint | The URL used to access your project and the data of your project. |True |
+    | Access key id | The ID  used to identify the user. | True |
+    | Access key | The key provided to you by Alibaba Cloud for authentication. | True |
+    | Project name | The name of your project in your Log Service used to isolate the resources of different users and control access to specific resources. | True |
+    | Logstore name | The unit in your Log Service that is used to collect, store, and query logs. | True |
+    | Query | The filter conditions in search statements used to obtain specific logs. Each query statement consists of a search statement and an analytic statement. The search statement and the analytic statement are separated with a vertical bar (\|).A search statement can be a keyword, a numeric value, a numeric value range, a space, or an asterisk . If you specify a space or an asterisk  as the search statement, no conditions are used for searching, and all logs are returned. <br />For example: * \| SELECT status, count AS PV GROUP BY status, will return all logs based on the analytic statement. | True |
```

Comment:
Hi, I think we should put as an example the default query that is set for the integration.
Query: *| select * from actiontrail_pa_trail
Description: This query retrieves all the events from the project set above.